8.1/startup/generateStartupCommand.js

```diff
@@ -47,7 +77,7 @@ if (!startupCommand) {
 if (!startupCommand) {
     console.log("No startup command or autodetected startup script " +
         "found. Running default static site.");
-    startupCommand = "node " + DEFAULTAPP;
+    startupCommand = nodeCommandPrefix + DEFAULTAPP;
```

Comment:
We could potentially skip initializing the AI runtime here, since it's just a "getting started" app. But, if the user enabled AI, they might expect to be able to see the HTTP traffic for the starter, in order to get a feel for the experience, before they push their own app code. Feedback would be appreciated here.